### ISSUE TITLE
TypeError: scrapePlaylist is not a function ?

### ISSUE THREAD

**Issue by JetSimon** [Aidurber/youtube-playlist-scraper#4]
I am getting ```TypeError: scrapePlaylist is not a function error.```

I have implemented just like the docs and have even tried ```scrapePlaylist.scrapePlaylist(id)``` just in case.

Using node.

```
const scrapePlaylist = require('youtube-playlist-scraper');

router.post('/convert', async (req,res) => {
  url = await req.body.youtubeLink;
  try{
    console.log(url);
    out = scrapePlaylist(url);
    console.log(out);
    res.send(out);
  }catch(e){
    console.log("YT ERROR: "+ e)
  }
});

**Comment by JetSimon** [comment]
Never mind I understand now! I can only use .default and the other functions. It literally wasn't a function.

You should update the readme? Or am I an idiot?

**Comment by Aidurber** [comment]
> You should update the readme? Or am I an idiot?

You're not the idiot here, it's me. I think I'll release a major version release where it's just a regular export so the usage would be: 
`const { scrapePlaylist } = require('youtube-playlist-scraper')`

Honestly I wasn't expecting people to use this package and since I use TypeScript, I failed to test commonJS 😬 I'll update the readme now mentioning to use `.default` whilst I cut another release. Thank you!

**Comment by Aidurber** [comment]
@JetSimon Fixed as of v2.0.0
